src/providers/discord.ts

```diff
@@ -16,8 +16,11 @@ export function discord(
     endpoints: {
       authorization: 'https://discord.com/api/oauth2/authorize',
       token: 'https://discord.com/api/oauth2/token',
-      userInfo: 'https://discord.com/api/users/@me'
+      userInfo: 'https://discord.com/api/users/@me',
+      logout: 'https://discord.com/api/oauth2/token/revoke' //TODO: add post method, because discord using the post method to logout
```

Comment:
Does this TODO mean that the logout as configured doesn't actually work because it must be a post request?

Reply:
Yes

Reply:
Then it's misleading and we shouldn't add this logout config. Keep the TODO, though!

Reply:
Fixed in new commit